docs/Drawer.md

```diff
@@ -44,6 +44,8 @@ const xsDown = useMediaQuery(theme.breakpoints.down('xs'));
 
 ### Drawer API
 
+Besides what's listed below, as an extension of [Material UI's `<Drawer>` API](https://material-ui.com/api/drawer/), the component will pass in all extra props.
```

Comment:
I would do these docs like Material UI (after the props table):
```
Any other props will be passed to the root element (**Material UI Drawer**)
```
![image](https://user-images.githubusercontent.com/29152776/80963246-8b714800-8ddc-11ea-8c64-9a899571f433.png)